Packs/NozomiNetworks/Integrations/NozomiNetworks/NozomiNetworks.py

```diff
@@ -0,0 +1,447 @@
+    readable_close_action = "closed_as_security" if close_action == "delete_rules" else "closed_as_change"
+    extracted_ids = argToList(args.get('ids'))
+    human_readable = f'Command changes the status of the following alerts: {extracted_ids} ' \
+        f'passed as "{readable_close_action}" in Nozomi Networks platform.'
+
+    client.http_post_request(
+        '/api/open/alerts/close',
+        {"ids": extracted_ids, "close_action": close_action})
+
+    return {
+        'readable_output': human_readable,
+        'outputs_prefix': None,
+        'outputs_key_field': None,
+        'outputs': None
+    }
+
+
+def has_last_id(lr):
+    return lr is not None and 'last_id' in lr
+
+
+def get_last_id(last_run):
+    result = None
+    if has_last_id(last_run):
+        result = f'{last_run.get("last_id", 0)}'
+    return result
+
+
+def filter_from_args(args):
+    if args and args.get('filter', '') != '':
+        filter = args.get('filter', '')
+        if '| where' in filter:
+            return filter
+        else:
+            return f" | where {filter}"
+    else:
+        return ''
+
+
+def assets_limit_from_args(args):
+    if args:
+        limit = int(args.get('limit', DEFAULT_ASSETS_FINDABLE_BY_A_COMMAND))
+        if limit > MAX_ASSETS_FINDABLE_BY_A_COMMAND:
+            return MAX_ASSETS_FINDABLE_BY_A_COMMAND
+        else:
+            return limit
+    else:
+        return DEFAULT_ASSETS_FINDABLE_BY_A_COMMAND
+
+
+def nodes_confirmed_filter(only_nodes_confirmed):
+    if only_nodes_confirmed and only_nodes_confirmed == 'True':
+        return ' | where mac_address:info.likelihood_level == confirmed'
+    else:
+        return ''
+
+
+def humanize_api_error(error):
+    if '401' in error:
+        return 'Authentication error, check your username and password'
+    else:
+        return error
+
+
+''' MAIN_FUNCTION '''
+
+
+def fetch_incidents(
+        client,
+        st=None,
+        last_run=None,
+        last_id=None,
+        risk=None,
+        fetch_also_n2os_incidents=None,
+        test_mode=False
+):
+    st = st or start_time(demisto.getLastRun(), demisto.params().get('fetchTime', '7 days').strip())
+    last_run = last_run or demisto.getLastRun()
+    last_id = last_id or get_last_id(demisto.getLastRun())
+    risk = risk or demisto.params().get('riskFrom', None)
+    fetch_also_n2os_incidents = fetch_also_n2os_incidents or demisto.params().get('fecthAlsoIncidents', False)
+
+    demisto_incidents, last_fetch, last_id_returned = \
+        incidents(st, last_id, last_run, risk, fetch_also_n2os_incidents, client)
+    demisto.setLastRun({'last_fetch': last_fetch, 'last_id': last_id_returned})
+
+    if not test_mode:
+        ack_alerts(nozomi_alerts_ids_from_demisto_incidents(demisto_incidents), client)
+        demisto.incidents(demisto_incidents)
+
+    return demisto_incidents, last_fetch
+
+
+def is_alive(client):
+    client = client or get_client()
+    error = None
```

Comment:
Any reason why `client` could be None?

```suggestion
def is_alive(client):
    error = None
```